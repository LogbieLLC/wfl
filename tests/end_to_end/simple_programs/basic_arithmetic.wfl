# Basic arithmetic operations test

define variable a = 10
define variable b = 5

# Addition
define variable sum = a + b
print sum  # Expected: 15

# Subtraction
define variable difference = a - b
print difference  # Expected: 5

# Multiplication
define variable product = a * b
print product  # Expected: 50

# Division
define variable quotient = a / b
print quotient  # Expected: 2

# Combined operations
define variable result = (a + b) * (a - b) / 2
print result  # Expected: 25 