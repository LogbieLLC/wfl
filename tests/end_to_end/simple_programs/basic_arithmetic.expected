15
5
50
2
25 